src/itemset.js

```diff
@@ -1,3 +1,16 @@
+/**
```

Comment:
Something I'd like to add but don't currently understand: why is the itemsetData attached to select questions an array, as in https://github.com/dimagi/Vellum/blob/master/tests/copy-paste.js#L688 ? Does vellum support multiple itemsets attached to a single question?